### PR TITLE
raise observable errors in sentry

### PR DESCRIPTION
**Pull request by sennett**
Makes it so we can see observable errors in sentry:

image

Also removes `useObservable` hook and renames `useObservableWithError` -> `useObservable`.

Also removes test pages that were used to test sentry errors.

### CODE REVIEW

File: components/vault/HistoryControl.tsx [hunk 1]
```diff
@@ -15,10 +15,8 @@ interface HistoryControlProps {
 
 export function HistoryControl({ generalManageVault }: HistoryControlProps) {
   const { vaultHistory$, vaultMultiplyHistory$ } = useAppContext()
-  const vaultHistoryWithError = useObservableWithError(
-    vaultHistory$(generalManageVault.state.vault.id),
-  )
-  const vaultMultiplyHistoryWithError = useObservableWithError(
+  const vaultHistoryWithError = useObservable(vaultHistory$(generalManageVault.state.vault.id))
```
Comment (peculiarity) [inline components/vault/HistoryControl.tsx:18]: I would rename these to `vaultHistory` and `vaultMultiplyHistory`

File: components/vault/ProtectionControl.tsx [hunk 2]
```diff
@@ -45,8 +45,8 @@ interface ProtectionControlProps {
 export function ProtectionControl({ vault, ilkData, account }: ProtectionControlProps) {
   const { automationTriggersData$, collateralPrices$ } = useAppContext()
   const autoTriggersData$ = automationTriggersData$(vault.id)
-  const automationTriggersDataWithError = useObservableWithError(autoTriggersData$)
-  const collateralPricesWithError = useObservableWithError(collateralPrices$)
+  const automationTriggersDataWithError = useObservable(autoTriggersData$)
+  const collateralPricesWithError = useObservable(collateralPrices$)
```
Comment (peculiarity) [inline components/vault/ProtectionControl.tsx:49]: Probably we can remove the `error` bit in here as well

File: components/vault/ProtectionControl.tsx [hunk 3]
```diff
@@ -45,15 +45,13 @@ interface ProtectionControlProps {
...
-    <WithErrorHandler
-      error={[automationTriggersDataWithError.error, collateralPricesWithError.error]}
-    >
+    <WithErrorHandler error={[automationTriggersData.error, collateralPrices.error]}>
       <WithLoadingIndicator
-        value={[automationTriggersDataWithError.value, collateralPricesWithError.value]}
+        value={[automationTriggersData.value, collateralPrices.value]}
         customLoader={<VaultContainerSpinner />}
       >
         {([automationTriggersData, collateralPrices]) => {
```
Comment (piekczyk) [inline components/vault/ProtectionControl.tsx:57]: doesn't lint screams here that automationTriggersData has been already declared somewhere?

File: components/vault/ProtectionControl.tsx (hunk 2 above)
Comment (sennett) [reply components/vault/ProtectionControl.tsx]: Done.

File: components/vault/HistoryControl.tsx (hunk 1 above)
Comment (sennett) [reply components/vault/HistoryControl.tsx]: Done

File: components/vault/ProtectionControl.tsx (hunk 3 above)
Comment (sennett) [reply components/vault/ProtectionControl.tsx:57]: Doesn't seem to no - lint doesn't seem to care about using the same variable name inside the closure.

Comment (sennett) [reply components/vault/ProtectionControl.tsx:57]: After some investigation it seems we don't have 'no-shadow' rule set.  This is what happens when we enable it for eslint 😁

image

**Review by ferni — DISMISSED** [review]

**Review by piekczyk — APPROVED** [review]

**Review by peculiarity — APPROVED** [review]

**Pull request merged**